components/atoms/AlphaBanner.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+/**
+ * Displays the Alpha AlphaBanner on the page
+ */
+
+export const AlphaBanner = ({ text }) => {
+  return (
+    <div className="bg-gray-100">
+      <div className="lg:container xxs:mx-0 xxs:px-0 lg:mx-auto lg:px-0 xxl:mx-auto">
+        <div className="xxs:w-screen lg:w-2/3 xl:w-1/2 p-4">
+          <div className="flex">
+            <span className="max-h-9 uppercase border-2 border-black p-1 tracking-widest ml-2">
```

Comment:
There is no letter spacing in the mockups 

This is what you have:

<img width="91" alt="Screen Shot 2021-04-15 at 12 16 25 PM" src="https://user-images.githubusercontent.com/26819992/114903061-68b92300-9de4-11eb-8bb2-828530b2de5e.png">

vs:

<img width="153" alt="Screen Shot 2021-04-15 at 12 17 12 PM" src="https://user-images.githubusercontent.com/26819992/114903170-85edf180-9de4-11eb-929d-f6f7a95c9cdc.png">

font size should be 16px and there should be 10px or so padding

Reply:
Without the letter spacing, the Alpha rectangle does not look the same. Probably due to the different font-family.

As for the font-size, the COVID 19 Benefit Finder page has a font-size of 14.76px. What should I follow? 

For the padding, I tried following the COVID 19 Benefit Finder as close as possible with tailwind.

Reply:
Thanks for your work so far Ben, made some additional changes to bring it a bit closer to the mockups here: https://www.figma.com/file/t5uDkPa86nsS3ssFESX2Em/BDM-DECD-Alpha-Site-Sprint-3?node-id=13%3A729